Makefile

```diff
@@ -332,5 +332,32 @@ logs:
 	socat pty,link=/tmp/ttyvnrf,waitslave tcp:127.0.0.1:8000 & disown
 	picocom /tmp/ttyvnrf -b 115200
 
-flash_with_gdb:
-	arm-none-eabi-gdb -e _build/nrf52832_xxaa_release.out -x flash.gdb
\ No newline at end of file
+daplink_erase_flash:
+	@echo Erase flash
+	openocd -f interface/cmsis-dap.cfg -f target/nrf52.cfg \
+  -c "init" -c "reset init" -c "nrf5 mass_erase" \
+  -c "reset" -c "exit"
+
+# flash the bluetooth stack using the DAPlink
+daplink_flash_softdevice: daplink_erase_flash
+	@echo Flashing: s132_nrf52_6.1.1_softdevice.hex

```

Comment:
Same as in #7, can you add in the README a link to where to download the softdevice file?

Reply:
done